src/Defaults/Carrier/Email.php

```diff
@@ -207,6 +207,12 @@ public function send(Triggerable $trigger)
 		}
 
 		$headers = [];
+
+		$fromHeader = $this->getFromHeaderFromSettings();
```

Comment:
Don't we have a problem. If there's a header from the settings AND defined in the carrier, we'll end up with array:

```
[
   'From settings',
   'From carrier'
]
```

Don't we?

Reply:
Yes, it will be in the array. But at the end when email is sending only the last element in the array matters. Because an email can only have one sender. I checked this in the headers of the sent message and there is only one sender.